sdk/src/main/java/ch/admin/bag/covidcertificate/sdk/android/repository/TrustListRepository.kt

```diff
@@ -124,37 +123,32 @@ internal class TrustListRepository(
 	private suspend fun refreshRevokedCertificates(forceRefresh: Boolean) = withContext(Dispatchers.IO) {
 		val shouldLoadRevokedCertificates = forceRefresh || !store.areRevokedCertificatesValid()
 		if (shouldLoadRevokedCertificates) {
-			val allRevokedCertificates = store.revokedCertificates?.revokedCerts?.toMutableList() ?: mutableListOf()
 			var since = store.revokedCertificatesSinceHeader
-			var validDuration: Long? = null
 
 			// Get the revocation list as long as the request is successful
-			var count = 0
 			var revocationListResponse = revocationService.getRevokedCertificates(getCacheControlParameter(forceRefresh), since)
 			while (revocationListResponse.isSuccessful) {
-				// Add all revoked certificates from the response to the list and save the validity duration of this response
-				allRevokedCertificates.addAll(revocationListResponse.body()?.revokedCerts.orEmpty())
-				validDuration = revocationListResponse.body()?.validDuration
+
```

Comment:
@M-Wong  can you review this change here please

Reply:
Looks good to me